README.md

```diff
@@ -1,54 +1,65 @@
+# docker_ucp
+
 [![Puppet
 Forge](http://img.shields.io/puppetforge/v/puppetlabs/docker_ucp.svg)](https://forge.puppetlabs.com/puppetlabs/docker_ucp)
 [![Build
 Status](https://travis-ci.org/puppetlabs/puppetlabs-docker_ucp.svg?branch=master)](https://travis-ci.org/puppetlabs/puppetlabs-docker_ucp)
 
-1. [Overview](#overview)
-2. [Module Description - What the module does and why it is useful](#module-description)
-3. [Setup - The basics of getting started with docker_ucp](#setup)
-3. [Usage - Configuration options and additional functionality](#setup)
-5. [Limitations - OS compatibility, etc.](#limitations)
-6. [Development - Guide for contributing to the module](#development)
+1.  [Description - What the module does and why it is useful](#description)
+2.  [Setup - The basics of getting started with docker_ucp](#setup)
+    -  [Beginning with docker_ucp](#beginning-with-docker-ucp)
+3.  [Usage - Configuration options and additional functionality](#usage)
```

Comment:
A Reference section is necessary, and it looks like the module is not compatible with Puppet Strings. This can be added in a follow-up PR.